packages/fxa-settings/src/pages/ResetPassword/index.test.tsx

```diff
@@ -273,70 +284,4 @@ describe('PageResetPassword', () => {
       ).not.toBeInTheDocument();
     });
   });
-
-  it('displays an error when the account is unknown', async () => {
```

Comment:
I think we could we still test the error response here by just manually emitting the error state from the provided handler? e.g. `beginResetHandler.mockRejectedValue(AuthUIError.THROTTLED)` should exercise this error state. 

To me this seems cleaner than relying on the mocks that are currently present in the container.tsx, and seems like a straight forward way to test the banner response, which has its logic held in this component not the container.

Reply:
Hmm, I guess after looking at the container more closely I see why the tests are done this way. See my question further down about about why we specifically  need a handler **result** reference. Is there a reason why we can't simply return the result from the handler?